CustomeDB/client.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#include <iostream>
+using namespace std;
+
+class client {
+
+    string name;
+    int id;
+    long phone;
+    string address;
+    int project_id;
+
+public:
+
+    string setname(string name);
+    int setid(int id);
+    int setphone(long phone);
```

Comment:
likewise, change phone setter to accept string. and setters should be void type.
change all the setters to void. and reflect changes to your .cpp file